### PR TITLE
Fixing misnumbered list items caused by inconsistent indentation

### PR DESCRIPTION
**Pull request by MALfunction84**
Fixing indentation of code blocks so that the numbered lists in which they appear are numbered correctly.

[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/102)

### CODE REVIEW

File: src/main/markdown/doc/latest/tutorial/manageevents.md [hunk 1]
```diff
@@ -129,36 +129,37 @@ For the enter key, you however have to use either KeyDownHandler or KeyUpHandler
...
-       * presses enter in the newSymbolTextBox.
-   */
-  private void addStock() {
-    // TODO Auto-generated method stub
-
-      }
-
-    }
-```
+   ```
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/manageevents.md]: the best is not to use triple backticks inside list. It's better to just indent the code with `4*(n+1)` spaces where n is the current depth of the list (so in this case 8 spaces)

File: src/main/markdown/doc/latest/tutorial/manageevents.md [hunk 2]
```diff
@@ -204,27 +205,27 @@ Finally, if the input is not valid, warn users via a dialog box.
...
-    newSymbolTextBox.setText("");
-
-    // TODO Don't add the stock if it's already in the table.
-    // TODO Add the stock to the table
-    // TODO Add a button to remove this stock from the table.
-    // TODO Get the stock price.
-
-}
-```
+   ```
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/manageevents.md]: idem here

File: src/main/markdown/doc/latest/tutorial/manageevents.md (hunk 1 above)
Comment (MALfunction84) [inline src/main/markdown/doc/latest/tutorial/manageevents.md]: Apologies. I did not see a consistent indentation in the file (4 for ordered lists, 3 for unordered, so nesting gets complicated). The takeaway is that the "Handling Keyboard Events" section currently renders two items numbered as "1" and renders an unordered list item as code.

That dangling curly brace (presumably for closing the class) complicates matters somewhat, but I will try to revise.

File: src/main/markdown/doc/latest/tutorial/manageevents.md [hunk 3]
```diff
@@ -80,40 +80,40 @@ import com.google.gwt.user.client.ui.VerticalPanel;
...
+    // Move cursor focus to the input box.
     newSymbolTextBox.setFocus(true);
 
-        // Listen for mouse events on the Add button.
+    // Listen for mouse events on the Add button.
     addStockButton.addClickHandler(new ClickHandler() {
       public void onClick(ClickEvent event) {
         addStock();
       }
     });
```
Comment (manolo) [inline src/main/markdown/doc/latest/tutorial/manageevents.md]: At the same point, do you mind to remove all these empty lines at the end of blocks.

Comment (MALfunction84) [inline src/main/markdown/doc/latest/tutorial/manageevents.md]: It wasn't completely clear what you were asking, but I tried to remove some blank lines.

Comment (manolo) [inline src/main/markdown/doc/latest/tutorial/manageevents.md]: That is, in GWT the convention is not to leave empty lines before closing-curly-brackets.

**Comment by manolo** [comment]
LGTM

**Pull request merged**